tools/samm-cli/src/main/java/org/eclipse/esmf/aas/AasListSubmodelsCommand.java

```diff
@@ -0,0 +1,48 @@
+package org.eclipse.esmf.aas;
+
+import java.io.File;
+import java.util.List;
+
+import org.apache.commons.io.FilenameUtils;
+import org.eclipse.esmf.AbstractCommand;
+import org.eclipse.esmf.LoggingMixin;
+import org.eclipse.esmf.aspectmodel.aas.AasToAspectModelGenerator;
+import org.eclipse.esmf.exception.CommandException;
+
+import picocli.CommandLine;
+
+@CommandLine.Command( name = AasListSubmodelsCommand.COMMAND_NAME, description = "Get list submodel templates of AAS input.",
+      descriptionHeading = "%n@|bold Description|@:%n%n",
+      parameterListHeading = "%n@|bold Parameters|@:%n",
+      optionListHeading = "%n@|bold Options|@:%n",
+      mixinStandardHelpOptions = true
+)
+public class AasListSubmodelsCommand extends AbstractCommand {
+   public static final String COMMAND_NAME = "list";
+
+   @CommandLine.Mixin
+   private LoggingMixin loggingMixin;
+
+   @CommandLine.ParentCommand
+   private AasCommand parentCommand;
+
+   @Override
+   public void run() {
+      final String path = parentCommand.getInput();
+      final String extension = FilenameUtils.getExtension( path );
+      if ( !extension.equals( "xml" ) && !extension.equals( "json" ) && !extension.equals( "aasx" ) ) {
```

Comment:
Not necessary, but only a suggestion:

```suggestion
      if ( Arrays.stream( AasFileFormat.values() ).anyMatch( format -> format.toString().equals( extension ) ) ) {
```